Assets/Scripts/JobSystem/JobsUI.cs

```diff
@@ -61,8 +64,8 @@ private GameObject InitialiseScheduleContainer()
         Image scheduleImage = scheduleObject.AddComponent<Image>();
         scheduleImage.sprite = containerBackgroundSprite;
 
-        scheduleObject.AddComponent<CanvasGroup>();
-        scheduleObject.AddComponent<Schedule>(); 
+        scheduleObject.AddComponent<CanvasGroup>().blocksRaycasts = true; 

```

Comment:
this is a hack if ive ever seen one 😂

Reply:
lol it's not exactly ideal, but will it suffice for now?

Reply:
yeah its fine, it was just funny